plugin/src/generators/code-checks/scripts.ts

```diff
@@ -1,5 +1,5 @@
 export default {
-  'lint': 'npx tsc && npx eslint ./',
+  'lint': 'npx tsc && ESLINT_USE_FLAT_CONFIG=false npx eslint ./',
```

Comment:
Please, use cross-env, like in other scripts with enviroment variables, so that it worked on Windows